script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -436,7 +436,7 @@ I feel a sense of relief as my vision returns to normal.
 -- Page 460, Offset 17716.
 -- 「ううん、一度見まわれば十分よ。気配っていうのはね、多少はその場所に残留するものなの。でも街には死者の気配がどこにもなかった。
 "Yeah, one look around is all we need.
- Traces of someone's presence linger around for a while, even after they're gone.
+ Traces of their presence linger around for a while, even after they're gone.
```

Comment:
The JP is a bit more vague here on presence, since it doesn't really attach a subject (literally approx "Presence is a thing which remains in a place."), which is why it was left as 'someone'.